be/src/olap/rowset/segment_group.cpp

```diff
@@ -723,6 +723,39 @@ OLAPStatus SegmentGroup::make_snapshot(const std::string& snapshot_path,
     return OLAP_SUCCESS;
 }
 
+OLAPStatus SegmentGroup::copy_files_to_path(const std::string& dest_path,
+                                            std::vector<std::string>* success_files) {
+    if (_empty) {
+        return OLAP_SUCCESS;
+    }
+    for (int segment_id = 0; segment_id < _num_segments; segment_id++) {
+        std::string dest_data_file = construct_data_file_path(dest_path, segment_id);
+        if (!check_dir_existed(dest_data_file)) {
```

Comment:
If exist file, then should check its length  or remove the exist file?

Reply:
I check dir exist or not firstly.